packages/gestalt/src/NewTabAccessibilityLabel.js

```diff
@@ -10,7 +10,9 @@ export default function NewTabAccessibilityLabel({
 |}): Node {
   const { accessibilityNewTabLabel } = useDefaultLabelContext('Link');
   return target === 'blank' ? (
-    <Box display="visuallyHidden">{`; ${accessibilityNewTabLabel}`}</Box>
+    <Box position="relative">
```

Comment:
Can we add a test for this, similar to the demo, so if this pops up as an issue with table, we can catch this error in case it become a bug again?

Reply:
I'll add test in Follow up